client/tracks/event.d.ts

```diff
@@ -14,6 +14,7 @@ export type Event =
 	| 'wcpay_welcome_learn_more'
 	| 'wcpay_stripe_connected'
 	| 'wcpay_connect_account_kyc_modal_opened'
+	| 'wcpay_csv_export_click'
```

Comment:
Do we need to remove older events from this TypeScript declaration (e.g. `wcpay_deposits_download`, `wcpay_disputes_download, etc) ?

Side note, maybe there's not a huge benefit in enforcing event names with TypeScript.

Reply:
Thanks for calling this out! I have removed the older events from the list.